code/dqe/rdbhdbcompare.q

```diff
@@ -0,0 +1,9 @@
+\d .dqe
+rdbhdbcompare:{[tab]
+  hdbmeta:(first exec w from .servers.getservers[`procname;`hdb1;()!();1b;0b])(meta;tab);
+  rdbmeta:(first exec w from .servers.getservers[`procname;`rdb1;()!();1b;0b])(meta;tab);
```

Comment:
what if we want to check `rdb1` against `hdb2`? or `rdb2` against `hdb2`? etc. Maybe procnames should be passed as args here?